Product/Production/Common/CONNECTCoreLib/src/test/java/gov/hhs/fha/nhinc/docrepository/adapter/AdapterComponentDocRepositoryOrchImplTest.java

```diff
@@ -84,28 +88,25 @@
     @Test
     public void testGetDocumentService() {
         AdapterComponentDocRepositoryOrchImpl docRepo = new AdapterComponentDocRepositoryOrchImpl();
-        assertTrue(docRepo.getDocumentService() instanceof DocumentService);
+        assertTrue(docRepo.getDocumentService() != null);
     }
 
     @Test
     public void testGetLargeFilesUtil() {
         AdapterComponentDocRepositoryOrchImpl docRepo = new AdapterComponentDocRepositoryOrchImpl();
-        assertTrue(docRepo.getLargeFileUtils() instanceof LargeFileUtils);
+        assertTrue(docRepo.getLargeFileUtils() != null);
     }
 
     @Test
     public void testGetDateUtil() {
         AdapterComponentDocRepositoryOrchImpl docRepo = new AdapterComponentDocRepositoryOrchImpl();
-        assertTrue(docRepo.getDateUtil() instanceof UTCDateUtil);
+        assertTrue(docRepo.getDateUtil() != null);
```

Comment:
check comments above

Reply:
Removed test, since it tests nothing - This getter only seems to exist for the sole purpose of being able to override the these methods to return mocked objects.